io_scene_godot/structures.py

```diff
@@ -453,6 +473,8 @@ def to_string(val):
         converter = CONVERSIONS.get(type(val))
         if converter is not None:
             val = converter(val)
+        elif val is None:
```

Comment:
If you really need this change, I think it better to put the `val is None` checking in front of all the other checkings

Reply:
fixed